src/utils/getTranslation.ts

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+const LABELS = {
+  "t.form.validation/required": "Required",
+  "t.form.validation/invalid-email": "Invalid email",
+  "t.form.validation/invalid-phone-number": "Invalid phone number",
+  "t.actions.errors/licenses-not-accepted":
+    "Terms and conditions were not accepted.",
+} as {
+  [p in string]: string;
+};
+
+export function getTranslation(key?: string): string | undefined {
+  return key ? LABELS[key] || key : undefined;
```

Comment:
Here you could consider printing console.error in case translation is not found. Its easier to find errors in console than spotting it on UI

Later when we integrate a error monitoring system with frontend, it could then send console errors to our system for us to catch these issues

Reply:
Great idea. Just did it.